docs/tutorials/pendulum.py

```diff
@@ -0,0 +1,122 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py:percent
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.16.1
+#   kernelspec:
+#     display_name: .venv
+#     language: python
+#     name: python3
+# ---
+
+# %% [markdown]
+# # Pendulum
+#
+# In this tutorial, we demonstrate how to generate data of a pendulum, and
+# introduce the mathematics of a pendulum.
+
+import math
```

Comment:
I think I found the reason for this.

Let's take the following lines as an example:
https://github.com/kausalflow/hamilflow/blob/a7426b090b94019a6fd50e176084be21e4e3e6f9/docs/tutorials/pendulum.py#L16-L27

percent script uses `# %%` to indicate code cells for whatever is coming, and `# %% [markdown]` for markdown cells.

Here in line 16, we indicate that the cell from line 17 to line 27 will be one single markdown cell.

To make sure percent script recognizes the code cells, we need to insert `# %%` before line 20.